src/map/script_constants.hpp

```diff
@@ -2371,6 +2387,23 @@
 	export_deprecated_constant2("SI_CHEERUP",992);
 	export_deprecated_constant2("SI_S_MANAPOTION",995);
 	export_deprecated_constant2("SI_M_DEFSCROLL",996);
+	
+	export_deprecated_constant2("SI_LIGHTOFMOON",1035);
+	export_deprecated_constant2("SI_LIGHTOFSUN",1036);
+	export_deprecated_constant2("SI_LIGHTOFSTAR",1037);
+	export_deprecated_constant2("SI_LUNARSTANCE",1038);
+	export_deprecated_constant2("SI_UNIVERSESTANCE",1039);
+	export_deprecated_constant2("SI_SUNSTANCE",1040);
+	export_deprecated_constant2("SI_FLASHKICK",1041);
+	export_deprecated_constant2("SI_NEWMOON",1042);
+	export_deprecated_constant2("SI_STARSTANCE",1043);
+	export_deprecated_constant2("SI_DIMENSION",1044);
+	export_deprecated_constant2("SI_DIMENSION1",1045);
+	export_deprecated_constant2("SI_DIMENSION2",1046);
+	export_deprecated_constant2("SI_CREATINGSTAR",1047);
+	export_deprecated_constant2("SI_FALLINGSTAR",1048);
+	export_deprecated_constant2("SI_NOVAEXPLOSING",1049);
+	export_deprecated_constant2("SI_GRAVITYCONTROL",1050);
```

Comment:
There's no point on adding the deprecated constant format, since these skills weren't implemented before the current format.

Reply:
Thanks, I'm going to remove it now.